ops/services/app_gateway/main.tf

```diff
@@ -235,12 +314,58 @@ resource "azurerm_application_gateway" "load_balancer" {
     target_url           = local.app_url
   }
 
+  rewrite_rule_set {
+    name = "simple-report-metabase-routing"
+
+    rewrite_rule {
+      name          = "metabase-wildcard"
+      rule_sequence = 100
+      condition {
+        ignore_case = true
+        negate      = false
+        pattern     = ".*metabase/(.*)"
+        variable    = "var_uri_path"
+      }
+
+      url {
+        path    = "/{var_uri_path_1}"
+        reroute = false
+        # Per documentation, we should be able to leave this pass-through out. See however
+        # https://github.com/terraform-providers/terraform-provider-azurerm/issues/11563
+        query_string = "{var_query_string}"
+      }
+    }
+  }
+
+  rewrite_rule_set {
+    name = "simple-report-staging-routing"
+
+    rewrite_rule {
+      name          = "staging-wildcard"
+      rule_sequence = 100
+      condition {
+        ignore_case = true
+        negate      = false
+        pattern     = ".*api/(.*)"
+        variable    = "var_uri_path"
+      }
+
+      url {
+        path    = "/{var_uri_path_1}"
+        reroute = false
+        # Per documentation, we should be able to leave this pass-through out. See however
+        # https://github.com/terraform-providers/terraform-provider-azurerm/issues/11563
+        query_string = "{var_query_string}"
+      }
+    }
+  }
+
   rewrite_rule_set {
     name = "simple-report-routing"
 
     rewrite_rule {
       name          = "api-wildcard"
-      rule_sequence = 100
+      rule_sequence = 101
```

Comment:
is there a link you have handy to what `rule_sequence` being set to `101` means as opposed to `100`? 🔍